WalletWasabi/Blockchain/Transactions/TransactionSummary.cs

```diff
@@ -6,16 +6,18 @@ namespace WalletWasabi.Blockchain.Transactions;
 
 public class TransactionSummary
 {
-	public TransactionSummary(SmartTransaction tx, Money amount)
+	public TransactionSummary(SmartTransaction tx, Money amount, Money fetchedFee)
 	{
 		Transaction = tx;
 		Amount = amount;
+		FetchedFee = fetchedFee;
 	}
 
 	public SmartTransaction Transaction { get; }
 	public Money Amount { get; set; }
+	public Money FetchedFee { get; set; } = Money.Zero;
```

Comment:
Init value is not necessary as the value is set in ctor.

Q: I can see `FetchedFee` cannot be null because even if we couldn't fetch it, `Money.Zero` is set.
 I am not sure which one is better, `null` or `Money.Zero` as null means it doesn't exist, Zero means it exists and zero.
 
Can't it cause confusion in some places?
TransactonTreeBuilder:
```
Fee = transactionSummary.GetFee() ?? transactionSummary.FetchedFee,
```

Fee is nullable in the UI but now It can never be null because Money.Zero will be set.

I am not sure, just asking.

Reply:
I try to avoid working with `null`, so I use `Money.Zero` instead. 
On the UI side, if the `Fee` is `Money.Zero` we don't show it to the user. (`IsFeeVisible is false`)

Not sure if it's the best to have this kind of logic on the UI side, but it helped me eliminate working with `null`, so I went with it.

Reply:
Took a deeper look now, I think it makes sense what you did. But from now `TransactionModel.Fee` doesn't have to be nullable. Can you fix it? It will also simplify a bit in some places.

Reply:
Sorry, Roland everything changed once again.
Now, we receive the effective FeeRate from the backend. I have to go through all the UI changes again and see if everything makes sense.